kafka/producer_test.go

```diff
@@ -176,6 +176,52 @@ func TestProducerAPIs(t *testing.T) {
 	if offsets != nil {
 		t.Errorf("OffsetsForTimes() failed but returned non-nil Offsets: %s\n", offsets)
 	}
+
+	// Purge messages that are not delivered to kafka
+	unreachableProducer, err := NewProducer(&ConfigMap{
+		"bootstrap.servers": "unreachable:9092",
```

Comment:
This might resolve and cause an outbound connection, instead use "127.0.0.1:65533"